.s2i/bin/assemble

```diff
@@ -30,7 +30,7 @@ cd /opt/app
 
 if compgen -G "/tmp/src/*.rockspec" > /dev/null; then
   cp /tmp/src/*.rockspec /opt/app/
-  find . -name "*.rockspec" -exec luarocks make {} \;
+  find . -name "*.rockspec" -exec luarocks build {} \;
```

Comment:
This will not use local filesystem and try to fetch them from git/registry. This is wrong for local rocks.